lib/ansible/plugins/filter/to_json.yml

```
@@ -0,0 +1,69 @@
+DOCUMENTATION:
+  name: to_json
+  author: core team
+  version_added: 'historical'
+  short_description: Convert variable to JSON string
+  description:
+    - Converts an Ansible variable into a JSON string representation.
+    - This filter functions as a wrapper to the Python C(json.dumps()) function.
```

```suggestion
    - This filter functions as a wrapper to the Python ``json.dumps()`` function.
```
Unfortunately C(...) cannot yet be used with nested round brackets.